### PR TITLE
Add CRUD for user

### PR DESCRIPTION
**Pull request by yaofeng-wang**

### CODE REVIEW

File: views/add_user.jade [hunk 1]
```diff
@@ -0,0 +1,25 @@
+extends layout
+
+block content
+    h1 #{title}
```
Comment (LuLechuan) [inline views/add_user.jade:4]: Add user is done by sign up right? Instead of the admin adding.

File: views/bids.jade [hunk 2]
```diff
@@ -2,12 +2,6 @@ extends layout
 
 block content
   h2 All bids
-<<<<<<< Updated upstream
```
Comment (LuLechuan) [inline views/bids.jade]: Please resolve your merge conflict

File: views/cars.jade [hunk 3]
```diff
@@ -0,0 +1,7 @@
+extends layout
+
+block content
+    ul
+        each car in cars
+            li
+                a(href="#") #{car.plate_number + " " +  car.driver + " " +  car.capacity + " " +  car.model}
```
Comment (LuLechuan) [inline views/cars.jade:7]: need a new line at the end of file

Comment (LuLechuan) [inline views/cars.jade:7]: Instead of a link, consider using a table to display the information of a car

File: views/rides.jade [hunk 4]
```diff
@@ -0,0 +1,8 @@
+extends layout
+
+block content
+  h2 All rides
+  ul
+    each ride in rides
+      li
+        a(href="#") #{ride.car + " "  + ride.start_time + " " + ride.source + " " + ride.destination + " " + ride.number_passenger + " " + ride.status}
```
Comment (LuLechuan) [inline views/rides.jade]: same as above, consider using tables.

**Review by LuLechuan — CHANGES_REQUESTED** [review]
Please have a look at the comments

File: views/add_user.jade (hunk 1 above)
Comment (yaofeng-wang) [reply views/add_user.jade:4]: In most circumstances, users will be added via sign up. But admin should also have the capability to add users?

Comment (LuLechuan) [reply views/add_user.jade:4]: yeah possible

**Pull request closed without merging**